### ISSUE TITLE
RUNNING_PID remains after ctrl+c

### ISSUE THREAD

**Issue by sirolf2009** [JPro-one/HelloJPro-Maven#3]
When I run the plugin using `mvn compile jpro:run` and I the press <kbd>CTRL</kbd>+<kbd>C</kbd>, the `RUNNING_PID` file will still be there. So the next time I try to run `mvn compile jpro:run` I'll get an error saying that an instance already exists or I should remove `RUNNING_PID`

**Comment by FlorianKirmaier** [comment]
Hi @sirolf2009 

Thank you for your bug report!
I cannot verify this issues on my system.
Can you tell me a little bit about your system?
Which OS are you using? Does this issue happen to you, in the unchanged HelloJPro-Project, or does it happen in your own project?

**Comment by ryppo** [comment]
Hi. I can confirm this under Win10.
@sirolf2009 Using jpro 2018.1.6 you can safely shut down the server process. Just place `mvn compile jpro:stop`. The server removes the file

**Comment by sirolf2009** [comment]
It happens in the HelloJPro project as well. 
I'm running linux
```
$ uname -a
Linux northpool2 4.9.0-8-amd64 #1 SMP Debian 4.9.110-3+deb9u6 (2018-10-08) x86_64 GNU/Linux
```
I've got another server that I tried this on, also a debian. It's got the same issue
```
$ uname -a
Linux pascal 3.16.0-4-amd64 #1 SMP Debian 3.16.51-3 (2017-12-13) x86_64 GNU/Linux
```
Would this method perhaps be helpful? https://docs.oracle.com/javase/10/docs/api/java/io/File.html#deleteOnExit()

**Comment by FlorianKirmaier** [comment]
@sirolf2009  @ryppo
I found out, that the Maven-Plugin behaved different as the Gradle-Plugin. This is fixed in 2018.1.8.
Previously, the maven-plugin didn't kill the old process, when `mvn compile jpro:run` was called.
Now, both plugin kills the old process, when it is still alive.

**Comment by ryppo** [comment]
@FlorianKirmaier 
Great. Just checked that. So we can remove our 

> stop.sh

 script from the server.
Thank you.
